src/clj/rems/api/services/workflow.clj

```diff
@@ -15,20 +15,28 @@
 (def ^:private validate-workflow-body
   (s/validator workflow/WorkflowBody))
 
+(defn invalid-forms-error [forms]
+  (let [invalid (seq (filter (comp nil? form/get-form-template :form/id) forms))]
```

Comment:
This is quite a mouthful even though it's basically right. Perhaps `invalid` should be named `invalid-forms`?

Reply:
Perhaps this could be written with `remove` and not `nil?` to simplify? (see the user one)

Reply:
Oh you did it in the later commits ... :+1: